Source/CarthageKit/Xcode.swift

```diff
@@ -177,6 +177,18 @@ private func <(lhs: ProjectEnumerationMatch, rhs: ProjectEnumerationMatch) -> Bo
 	return lhs.locator < rhs.locator
 }
 
+/// A shared scheme defined in a project or workspace
+private struct SharedScheme {
+	let schemeName: String
+	let project: ProjectLocator
+}
```

Comment:
I don't really like this struct. Since Xcode projects contain schemes, it doesn't seem right for `SharedScheme` to have a `ProjectLocator`. Maybe instead we could use a named tuple?